apps/settings/js/modules/settings_panel.js

```diff
@@ -89,10 +96,16 @@ define(function(require) {
 
           _panel = panel;
           PanelUtils.activate(panel);
+          if (SpatialNavigationHelper.isEnabled()) {
+            window.addEventListener('keyup', _goBackHandler.bind(this));
+          }
 
           return options.onInit(panel, initOptions);
         },
         onUninit: function() {
+          if (SpatialNavigationHelper.isEnabled()) {
+            window.removeEventListener('keyup', _goBackHandler.bind(this));
```

Comment:
this wont work because `_goBackHandler.bind(this)` will create a new object, please create `boundGoBackHandler = _goBackHandler.bind(this)` and call it instead